src/Altinn.Platform/Altinn.Platform.Storage/Storage/Configuration/AzureStorageConfiguration.cs

```diff
@@ -29,5 +29,25 @@ public class AzureStorageConfiguration
         /// url for the blob end point
         /// </summary>
         public string BlobEndPoint { get; set; }
+
+        /// <summary>
+        /// url for the app owner Key Vault
+        /// </summary>
+        public string OrgKeyVaultURI { get; set; }
+
+        /// <summary>
+        /// name of app owner storage account
+        /// </summary>
+        public string OrgStorageAccount { get; set; }
+
+        /// <summary>
+        /// name of SAS definition in app owner Key Vault
+        /// </summary>
+        public string OrgSasDefinition { get; set; }
+
+        /// <summary>
+        /// name of storage container in app owner storage account
+        /// </summary>
+        public string OrgStorageContainer { get; set; }
```

Comment:
Do we need the StorageContainer variable if this is introduced?

Reply:
we don't _need_ it, it is used for potential fail over if the application owner doesn't have local storage configured, or connection to the local storage fails.